### ISSUE TITLE
When will the code be available?

### ISSUE THREAD

**Issue by Runsong123** [yashbhalgat/Contrastive-Lift#1]
Dear author,

Thanks for your great work.
I want to test your methods in my experiments and I want to know when the code will be available? Besides, Does the proposed method take the similar time as panoptic-lifting for optimization time on ScanNet. BTW, I found the panoptic-lifting takes 36 hours on my single 3090 RTX for one ScanNet scene (042302).
Looking forward to your reply. Many thanks in advance.

Bests,
Runsong

**Comment by yashbhalgat** [comment]
Hi Runsong, thank you for your interest in my work! I will be releasing the code over the weekend along with some pretrained checkpoints. Stay tuned!

The loss functions used by Contrastive-Lift are the same as Panoptic-Lifting, except of course the instance branch loss. And the most time-consuming loss is the "segment consistency" loss, which is present in both methods (ours and PanopLi). I found that you don't need train the model for as many epochs as PanopLi, so the training can be done in around 20-24 hours for Contrastive-Lift. Hope this answers your question!

**Comment by Runsong123** [comment]
I see. Thank you very much for your explanation !!
